### DISCUSSION TITLE
New Player - Server Timezone

### DISCUSSION THREAD

**Discussion by cazous** [chromiecraft/chromiecraft#7173] [General]
Hello, everyone.

I'm a new player living in Brazil and was wondering if there are any NA times raids/guils in the server.

**Comment by BeefSteaker** [comment]
Your best bet would be looking through guilds-recruiting on the discord. 
Rum rage, Auts, Angels of Azeroth all have NA time raids, probably more too

**Comment by IskrenPlashkov** [comment]
If alongside portuguese you speak spanish you can try La Mano De Plata too.
